package.json

```diff
@@ -1,42 +1,58 @@
-    "test-debug": "truffle compile && DEBUG=app:*,test:* truffle test",
+    "test": "truffle test test/*.js",
+    "test-debug": "DEBUG=app:*,test:* truffle test test/*.js",
+    "rpc": "ganache-cli -d",
+    "networks-extract": "npx truffle networks --clean && npx tnt extractNetworks",
+    "networks-inject": "npx tnt injectNetworks",
+    "networks-reset": "truffle networks --clean && npm run networks-inject",
+    "compile": "truffle compile",
     "migrate": "truffle migrate",
-    "rpc": "cross-conf-env ganache-cli --gasLimit npm_package_config_gasLimit --account=\"0x0191ecbd1b150b8a3c27c27010ba51b45521689611e669109e034fd66ae69621,9999999999999999999999999999999999999999999\" --account=\"0x00f360233e89c65970a41d4a85990ec6669526b2230e867c352130151453180d,9999999999999999999999999999999999999999999\" --account=\"0x987a26abca7432016104ce2f24ce639340e25afe06ac69f68791399e7a5d1028,9999999999999999999999999999999999999999999\"    --account=\"0x89af34b1b7347834048b99423dad174a14bf14540d720d72c16ae92e94b987cb,9999999999999999999999999999999999999999999\" --account=\"0xc867be647eb2bc51e4c0d61066859875cf3634fe949b6f5f85c69ab90e485b37,9999999999999999999999999999999999999999999\" --account=\"0xefabcc2377dee5e51b5a9e65a3854aec85fbbec3cb584d8ad4f9679869fb33c6,9999999999999999999999999999999999999999999\"",
-    "truffle": "truffle",
+    "preversion": "npm run restore",
+    "restore": "rm -rf build && npm run compile && npm run networks-reset",
+    "networks": "truffle networks",
     "lint": "eslint .",
-    "solium": "solium --dir ./contracts/"
+    "solium": "solium --dir ./contracts/",
+    "get-0x-list": "truffle exec src/scripts/get-0x-list.js",
+    "get-cryptocompare-tokens": "node src/scripts/get-cryptocompare-tokens",
+    "get-token-prices": "node src/scripts/get-token-prices.js",
+    "update-price-oracle": "truffle exec src/scripts/update-price-oracle.js"
   },
   "license": "ISC",
   "devDependencies": {
-    "eslint": "^5.4.0",
-    "eslint-config-standard": "^11.0.0",
+    "@gnosis.pm/truffle-nice-tools": "^1.2.1",
+    "babel-eslint": "^10.0.1",
+    "dotenv": "^6.1.0",
+    "eslint": "^5.9.0",
+    "eslint-config-standard": "^12.0.0",
     "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-node": "^8.0.0",
     "eslint-plugin-promise": "^4.0.0",
-    "eslint-plugin-standard": "^3.1.0",
-    "babel-eslint": "^8.2.1",
-    "bignumber.js": "^5.0.0",
-    "truffle": "^4.1.13",
+    "eslint-plugin-standard": "^4.0.0",
+    "fractional": "^1.0.0",
+    "got": "^9.3.2",
+    "moment-timezone": "^0.5.23",
     "solium": "^1.1.8",
-    "cross-conf-env": "^1.1.2",
-    "solc": "^0.4.24",
-    "solc-cli": "^0.3.0",
-    "ganache-cli": "^6.1.6"
+    "truffle": "^5.0.0-beta.2",
+    "yargs": "^12.0.5"
   },
   "dependencies": {
-    "@daostack/arc": "^0.0.0-alpha.52",
-    "@gnosis.pm/dx-contracts": "^1.0.3",
-    "debug": "^3.1.0",
+    "@daostack/arc": "0.0.0-alpha.56",
```

Comment:
use arc 0.0.0-alpha.58

Reply:
We will update in another branch as a new PR as soon as you approve this one.